docs/reference/di/providing-different-implementation-of-a-service.md

```diff
@@ -42,7 +42,7 @@ object MainApp extends ZIOAppDefault {
       _        <- ZIO.debug(s"Application started with $poolSize pool size.")
     } yield ()
 
-  def run = myApp.provideSome(AppConfig.appArgsLayer)
+  def run = myApp.provideLayer(AppConfig.appArgsLayer)
```

Comment:
```suggestion
  def run = myApp.provideSome[ZIOAppArgs](AppConfig.appArgsLayer)
```

Reply:
Thank you! I updated my commit to reflect your suggestion.